### PR TITLE
Add player glow to last player on team (configurable)

### PR DESCRIPTION
**Pull request by minesettimi**
If the convar is enabled the last player on a team (bosses excluded) is highlighted to make it easier to move the end of the round along especially if the last player is hiding from the boss.

### CODE REVIEW

File: addons/sourcemod/scripting/freak_fortress_2/gamemode.sp [hunk 1]
```diff
@@ -955,8 +955,13 @@ void Gamemode_PlayerRunCmd(int client)
 			if(PlayersAlive[team] < 3)
 				TF2_AddCondition(client, TF2_GetPlayerClass(client) == TFClass_Scout ? TFCond_Buffed : TFCond_CritCola, 0.5);
 			
-			if(PlayersAlive[team] < 2)
+			if(PlayersAlive[team] < 2) {
```
Comment (Batfoxkid) [inline addons/sourcemod/scripting/freak_fortress_2/gamemode.sp:958]: Change this to match the rest of the code format with

```c
if(statement)
{
```

**Review by Batfoxkid — CHANGES_REQUESTED** [review]
Is it on purpose that bosses aren't affected by the outline?

**Comment by minesettimi** [comment]
Sort of, I felt it wasn't as necessary for the boss but also I cant seem to find a way to implement it.

**Review by Batfoxkid — APPROVED** [review]
> Sort of, I felt it wasn't as necessary for the boss but also I cant seem to find a way to implement it.

This could possibly be added in the future, I'll take this as it is now.

**Pull request merged**